samples/sensor/mpu6050/README.rst

```diff
@@ -0,0 +1,59 @@
+.. _mpu6050:
+
+MPU6050: Invensense Motion Tracking Device
+##########################################
+
+Description
+***********
+
+This sample application periodically (0.5 Hz) measures samples the
```

Comment:
Use either ``samples`` or ``measures``, but not both words.  Since you say "samples" below, I'd change this then to say just ``samples`` too.

Reply:
I went with measures/measurements to avoid confusion with the fact this is a sample application.